client/src/pages/Settings/Settings.tsx

```diff
@@ -1,12 +1,17 @@
 import { useState } from 'react';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
 import { Box, Typography, Tabs, Tab } from '@material-ui/core';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import Payment from '../../components/Payment/Payment';
 import useStyles from "./useStyles";
 
 interface TabPanelProps {
   children?: React.ReactNode;
   index: number;
   value: number;
 }
+const stripeKey = loadStripe('pk_test_51JEJJrK3QVx74yV9vYujCrOpA8KwLKEE7uKSjHmUO0GGxUYSa5uxMlgaCZAbflwU9U0Npb2k4a86JEJBGmuBOfbw00CU3LpL8W' || '');
```

Comment:
What is the reason behind using a logical OR here? :)

Reply:
I blame stack over flow :)

Reply:
You can exclude it :)